### DISCUSSION TITLE
科研/学术工作者使用Obsidian的最佳实践？

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#19] [实用技巧/最佳实践]
日常挖坑，比如
* 跟zotero连用：https://forum.obsidian.md/t/zotero-integrations/91
* 引用插件：citations （插件中心中搜索），使用指引： #28

**Comment by makerjackie** [comment]
关于数学公式：
obsidian是支持的，输入下面的，然后ctrl + E 预览一下试试
```
$$ E=mc^2 $$
```

配合以下内容食用更佳
* https://www.latexlive.com/ （latex公式识别 以及 快速打出你要的公式）
* latex语法：https://www.latexlive.com/help#d17

**Comment by Quorafind** [comment]
山鸡的 Zotero 与 Ob 连用：https://github.com/obsidianzh/forum/discussions/28
